ci/run-tests.py

```diff
@@ -65,14 +65,12 @@ def change_setting(content, setting, value):
 shutil.copyfile(init_txt_path, init_txt_path + '.orig')
 with open(init_txt_path) as f:
     init_contents = f.read()
-init_contents = change_setting(init_contents, 'INTRO', 'NO')
 init_contents = change_setting(init_contents, 'SOUND', 'NO')
 init_contents = change_setting(init_contents, 'WINDOWED', 'YES')
-init_contents = change_setting(init_contents, 'WINDOWEDX', '80')
-init_contents = change_setting(init_contents, 'WINDOWEDY', '25')
-init_contents = change_setting(init_contents, 'FPS', 'YES')
-if args.headless:
-    init_contents = change_setting(init_contents, 'PRINT_MODE', 'TEXT')
+init_contents = change_setting(init_contents, 'WINDOWEDX', '1200')
+init_contents = change_setting(init_contents, 'WINDOWEDY', '800')
+#if args.headless:
+#    init_contents = change_setting(init_contents, 'PRINT_MODE', 'TEXT')
```

Comment:
commented out since TEXT mode is no longer supported -- but may be supported again in the future.
in the meantime, we launch with Xfrb to provide an X screen on which to run the graphical window